### ISSUE TITLE
Something wrong when training with voxelnet_late_fusion.yaml and second_late_fusion.yaml?

### ISSUE THREAD

**Issue by lubin202209** [DerrickXuNu/OpenCOOD#130]
Hello, I encountered with some strange problems when traing with voxelnet_late_fusion.yaml and second_late_fusion.yaml，I found that when train with batch_size of 1, everything works fine for both training and inferencing. However, when I use a batch_size bigger than 1 (for example, 2) for training, I get the following error when calculating the loss on the validation set:

![d40f89e98bf4f8f33045efa7bf1bda7](https://github.com/DerrickXuNu/OpenCOOD/assets/116709464/933ac5ce-6eb9-4089-93f3-f2d3980fcd4f)

As you can seen, I printed   ouput_dict['psm'].shape，ouput_dict['rm'].shape，batch_data['ego']['label_dict']['targets'].shape，batch_data['ego']['label_dict']['pos_equal_one'].shape

![d0e0ef17db3ba4db477ec489370ae3e](https://github.com/DerrickXuNu/OpenCOOD/assets/116709464/3e16f7a3-cd33-4bce-abed-72f1612ac70f)

I found that the output of the model is based on batch_size as 2, but the true value in batch_data is 1，It seems that there is a mismatch in the data loading part of LatefusionDataset. Can you help me solve this problem? How can I fix it?
